src/App.js

```diff
@@ -1,19 +1,28 @@
 import React from "react";
 import { Route, Routes } from "react-router-dom";
+import { useState } from "react";
 import ReviewList from "./Componenets/ReviewList";
 import Header from "./Componenets/Header";
 import NavBar from "./Componenets/navBar";
 import Categories from "./Componenets/Categories";
 import SingleReview from "./Componenets/SingleReview";
 import Comments from "./Componenets/Comments";
 const App = () => {
+  const [selectedCategory, setSelectedCategory] = useState("");
+
+  const handleCategorySelection = (selectedCategory) => {
+    setSelectedCategory(selectedCategory);
+  };
   return (
     <div>
       <Header />
       <NavBar />
-      <Categories />
+      <Categories handleCategorySelection={handleCategorySelection} />
       <Routes>
-        <Route path="/" element={<ReviewList />} />
+        <Route
```

Comment:
For this ticket, we need each to be able to view a separate page for each category, i.e. each category should have their own url. For example I should be able to send www.yourwebsite.com/reviews/dexterity to a friend and they will see a page of the reviews sorted by category.
Think about how you can use a parametric endpoint and `useParams()` to achieve this. You may need to also use the navigate function in the Navigating Programmatically section of the notes, triggered by your submit button https://notes.northcoders.com/courses/js-front-end/react-router